cli/kata-env.go

```diff
@@ -213,6 +215,7 @@ func getHostInfo() (HostInfo, error) {
 		Distro:             hostDistro,
 		CPU:                hostCPU,
 		VMContainerCapable: hostVMContainerCapable,
+		SupportVSocks:      vcUtils.SupportsVsocks(),
```

Comment:
looks good @devimc. But I think you need to bump the format version:
https://github.com/kata-containers/runtime/blob/6566d9102c2a5727e1509f3446f23d213412845c/cli/kata-env.go#L29